tests/api/api/test_feature_sets.py

```diff
@@ -0,0 +1,101 @@
+from http import HTTPStatus
+from uuid import uuid4
+
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+
+fs = {
+    "metadata": {
+        "name": "dummy",
+        "labels": {"owner": "saarc", "group": "dev"},
+        "tag": "latest",
+    },
+    "spec": {
+        "entities": [{"name": "ticker", "value_type": "str"}],
+        "features": [
+            {"name": "time", "value_type": "datetime"},
+            {"name": "bid", "value_type": "float"},
+            {"name": "ask", "value_type": "time"},
+        ],
+    },
+    "status": {
+        "state": "created",
+        "stats": {
+            "time": {"count": "8", "unique": "7", "top": "2016-05-25 13:30:00.222222"}
+        },
+    },
+}
+
```

Comment:
(Some nit picking) Putting a general comment (relevant to all test files), please do a pass and change to full names, e.g. `resp` -> `response`, `proj` -> `project`

Reply:
Another general comment  (relevant to all test files), when you do `assert X == Y` you don't have to add the string in the end e.g.
`assert X == Y, "blabla"` this string is there in order to make debugging easier when tests fail, but putting there some strings without meaning ("list") is redundant.
I'm totally fine with not putting these, if you feel like you do want to use them, mostly it's helpful to format into the string values that are not part of the assertions

Reply:
Another general comment, use our logger instead of `print`

Reply:
TBH, I don't really think I need those `print`s, they were just there for my testing. I'll remove them.
Will correct the rest.